<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageinicio</class>
 <widget class="QWidget" name="imageinicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">imagen (:/Fondo2.png)</string>
  </property>
  <widget class="QLabel" name="imagen">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pregunta">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>200</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gill Sans Ultra Bold&quot;;</string>
   </property>
   <property name="text">
    <string>Encuentre el animal</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="opcion1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="opcion2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="opcion3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
